app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -3619,6 +3619,118 @@ class StateFragmentTest {
     }
   }
 
+  @Test
+  fun testStateFragment_interactions_noRadioItemSelected_defaultSelectionTextIsDisplayed() {
+    launchForExploration(TEST_EXPLORATION_ID_2, shouldSavePartialProgress = false).use {
+      startPlayingExploration()
+      playThroughPrototypeState1()
+      playThroughPrototypeState2()
+      playThroughPrototypeState3()
+      playThroughPrototypeState4()
+
+      verifyViewTypeIsPresent(SELECTION_INTERACTION)
+      onView(withId(R.id.selection_interaction_textview))
+        .check(matches(withText("Please select all correct choices.")))
+    }
+  }
+
+  @Test
+  fun testStateFragment_interactions_oneRadioItemSelected_selectionTextIsDisplayed() {
+    launchForExploration(TEST_EXPLORATION_ID_2, shouldSavePartialProgress = false).use {
+      startPlayingExploration()
+      playThroughPrototypeState1()
+      playThroughPrototypeState2()
+      playThroughPrototypeState3()
+      playThroughPrototypeState4()
+
+      verifyViewTypeIsPresent(SELECTION_INTERACTION)
+      selectItemSelectionCheckbox(optionPosition = 0, expectedOptionText = "Red")
+
+      onView(
+        withId(
+          R.id.selection_interaction_textview
+        )
+      ).check(matches(withText("You may select more choices.")))
```

Comment:
```suggestion
      onView(withId(R.id.selection_interaction_textview))
        .check(matches(withText("You may select more choices.")))
```

Cleaner and more compact way to wrap. Ditto elsewhere (where applicable).

Reply:
okay